src/components/Profile/Profile.jsx

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import s from "./Profile.module.css";
+import MyPosts from "./MyPosts/MyPosts";
+
+const Profile = () => {
+  return (
+    <div className={s.content}>
+      <div>
+        <img src="https://animal-wallpaper.com/wallpaper/minimalist-background-hd-For-Background-HD-Wallpaper.jpg" />
```

Comment:
Consider putting this string url into a constant variable in the current file and using the variable reference here.

Reply:
complite